### PR TITLE
fix size comparison workflow

### PR DESCRIPTION
**Pull request by matt-fidd**
This fixes the errors we're seeing with the size comparison workflow

eg.
https://github.com/actualbudget/actual/actions/runs/11219565964/job/31185802750?pr=3581

new default value for `allow_forks` is `false` as of dawidd6/action-download-artifact v4
https://github.com/dawidd6/action-download-artifact/releases/tag/v4

### CODE REVIEW

**Comment by matt-fidd** [comment]
I don't believe the test will pass in this PR (the same way it didn't fail in #3577) because it runs on `pull_request_target`.

File: .github/workflows/size-compare.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
           workflow: build.yml
           name: build-stats
           path: head
+          allow_forks: true
```
[the remark this answers is not in the document]
Comment (matt-fidd) [reply .github/workflows/size-compare.yml:64]: Fixed, it needed rebasing onto master

**Review by youngcw — APPROVED** [review]
Looks good to me.  Lets give it a shot an see if things are fixed

**Comment by matt-fidd** [comment]
Testing in https://github.com/actualbudget/actual/pull/3581

**Pull request merged**